### PR TITLE
fix: plugin refactor v1

### PR DESCRIPTION
**Pull request by marcgarreau**
#### What does it do?
First of probably several refactors to bring all prior naming iterations (e.g. nodes, clients, services) to `plugin`. This focuses on renaming `Nodes` directory to `Plugins` and `client` Redux directory to `plugin` (+ their resulting ripples).

Note: very many `client` (and other noun) references within. Not planning on addressing all of them until part 2 or 3.
#### Does it close any issues?
Partially addresses https://github.com/ethereum/grid/issues/394

### CODE REVIEW

File: src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js [hunk 1]
```diff
@@ -174,10 +174,10 @@ class DynamicConfigFormItem extends Component {
 }
 
 function mapStateToProps(state, ownProps) {
-  const selectedClient = state.client.selected
+  const selectedClient = state.plugin.selected
```
Comment (ryanio) [inline src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js]: `const selectedPlugin`?

File: src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js [hunk 2]
```diff
@@ -174,10 +174,10 @@ class DynamicConfigFormItem extends Component {
...
 
   return {
-    itemValue: state.client[selectedClient].config[ownProps.itemKey]
+    itemValue: state.plugin[selectedClient].config[ownProps.itemKey]
```
Comment (ryanio) [inline src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js]: `selectedPlugin`

File: src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js (hunk 1 above)
Comment (marcgarreau) [reply src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js]: (to recap discord convo:) reserving for part 2; many `client` references still within.

File: src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js (hunk 2 above)
Comment (marcgarreau) [reply src/components/Plugins/ClientConfig/DynamicConfigForm/FormItem.js]: (same)

**Pull request merged**